index.bs

```diff
@@ -1307,8 +1307,15 @@ All USB devices MUST have a <a>default control pipe</a> which is
         with a {{NotFoundError}} and abort these steps.
     1. If <a>this</a>.{{USBDevice/[[claimedInterface]]}}[|interfaceIndex|] is <code>true</code>,
         <a>resolve</a> |promise| and abort these steps.
-    1. If |interfaces|[|interfaceIndex|].{{USBInterface/[[isProtectedClass]]}} is <code>true</code>,
-        [=reject=] |promise| with a {{SecurityError}} and abort these steps.
+    1. Let |unrestricted| be <code>false</code>.
```

Comment:
As a follow-up, we should actually specify how the blocklist is checked and add the same check for the `usb-unrestricted` permission policy there.

Reply:
I've updated this PR to integrate with the blocklist, PTAL